src/plays/wordle/styles.css

```diff
@@ -0,0 +1,158 @@
+/* enter stlyes here */
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
```

Comment:
@SamirMishra27 Can you please use this class name `wordle-game-body` to prefix all the styles in this file?

Reply:
I'm confused, I will remove `:root selector` from the file?

Reply:
Done!